Content.Server/Medical/MedicalScannerSystem.cs

```diff
@@ -78,29 +78,27 @@ private void OnRelayMovement(EntityUid uid, MedicalScannerComponent scannerCompo
 
         private void AddInsertOtherVerb(EntityUid uid, MedicalScannerComponent component, GetVerbsEvent<InteractionVerb> args)
         {
-            if (args.Using == null ||
-                !args.CanAccess ||
-                !args.CanInteract ||
-                IsOccupied(component) ||
-                !CanScannerInsert(uid, args.Using.Value, component))
+            if (args.Using == null
+                || !args.CanAccess
+                || !args.CanInteract
+                || IsOccupied(component)
+                || !CanScannerInsert(uid, args.Using.Value, component)
+                || !TryComp<MetaDataComponent>(args.Using.Value, out var metadata))
```

Comment:
This shouldn't be necessary- you can use `MetaData(ent)` to grab the MetaDataComponent of any Entity, same as Transform.

Reply:
Will do!